src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-tutorial-java.mdx

```diff
@@ -223,9 +223,7 @@ To monitor our demo app with the OpenTelemetry Java agent:
 
 ### Set up the demo app manually [#manual-instrum]
 
-In this track, you'll roll up your sleeves and tinker with the engine of the car. This is the approach to take if you want to have the most control over what telemetry is reported and want to see details about how it's done.
-
-You'll manually insert instrumentation into our demo app to capture telemetry, and you'll configure the SDK to export that data to New Relic.
+The previous tutorial helped you explore automatic instrumentation with the OpenTelemetry Java agent. This tutorial will show you how to use custom instrumentation to have more control over the telemetry you gather. You'll manually insert instrumentation into our demo app to capture telemetry, and then you'll configure the SDK to export that data to New Relic.
```

Comment:
Are the three tabs meant to be sequential? Referring to previous tutorials makes it seem that way. 

- If they aren't sequential, I might tweak this language very slightly. "This tutorial shows you how to use custom instrumentation blahblahblah. If you have X use case, then follow the instructions in [other tabs]" 

- If it is sequential, I wonder if the horizontal tab component is the best one to use. If it is, maybe we should at least preface the titles with "Step 1" etc

Reply:
The confusion came in part because our horizontal tabs should be used as parallel/discrete procedures with no overlap between the other tabs. A user should be able to work through one without needing context or to reference what the others do. The 'previous tutorial helped you' implies that the reader worked through the other procedures and was helped by them, and moved on to this next one. 

This tweak is fine tho, assuming a user reads the doc top to bottom very carefully, and remembers it. I do think referencing other tutorials can introduce confusion, however.